addons/l10n_be/i18n/de.po

```diff
@@ -367,7 +273,7 @@ msgstr "Buchungssatz"
 #. module: l10n_be
 #: model:account.report.line,name:l10n_be.tax_report_title_operations
 msgid "Operations"
-msgstr "Operationen"
+msgstr "Umsätze"
```

Comment:
Does this have the same meaning? It's intended for both sales and purchases (Operations).

Reply:
No idea, I merely reused the translations from the Transifex project. If it's not accurate, you might want to fix it on Transifex